test/Test/GenericsRep.purs

```diff
@@ -132,9 +132,7 @@ instance booleanAlgebraB1 :: BooleanAlgebra B1
 
 genericsRepTests :: AlmostEff
 genericsRepTests = do
-  -- Temporarily comment this out to show that CI works
-  -- I'm not sure what this should be
-  -- assert "not sure what this should be" $ show (cons 1 (cons 2 Nil)) == "Cons(1, Cons(2, Nil))"
+  assert "not sure what this should be" $ show (cons 1 (cons 2 Nil)) == "Cons(1, Cons(2, Nil))"
```

Comment:
What should this be? And are there other things that could be affecting this test?